bundles/org.openhab.binding.mqtt.generic/src/main/java/org/openhab/binding/mqtt/generic/tools/WaitForTopicValue.java

```diff
@@ -24,14 +25,14 @@
 import org.eclipse.smarthome.io.transport.mqtt.MqttMessageSubscriber;
 
 /**
- * Waits for a topic value to appear on a MQTT topic. One-time useable only per instance.
+ * Waits for a topic value to appear on a MQTT topic. One-time usable only per instance.
  *
  * @author David Graeff - Initial contribution
  */
 @NonNullByDefault
 public class WaitForTopicValue {
-    private CompletableFuture<String> future = new CompletableFuture<>();
-    private final CompletableFuture<Boolean> subscripeFuture;
+    private final CompletableFuture<String> future = new CompletableFuture<>();
```

Comment:
I don't think this needs to be a field anymore since it isn't used outside of the constructor.